src/app/common/duck/utils.ts

```diff
@@ -40,13 +41,20 @@ export const handleSelfSignedCertError = (failedUrl: string, dispatch: any) => {
 };
 
 const DNS1123Error = (value) => {
-  return `Invalid character: "${value}". Plan names must be DNS-1123 label compliant, starting and
-    ending with a lowercase alphanumeric character and containing only lowercase alphanumeric
-    characters, "." or "-".`;
+  return `Invalid character: "${value}". Must be DNS-1123 label compliant, starting and ending with
```

Comment:
I don't think this language change is necessary here.

Reply:
Actually `DNS1123Error` isn't specific to Plan name. The error message should be more generic. That was changed in https://github.com/konveyor/mig-ui/pull/814 (by me but I missed that).